config.json

```diff
@@ -23,9 +23,52 @@
   },
   "exercises": {
     "concept": [],
-    "practice": []
+    "practice": [
+      {
+        "slug": "hello-world",
+        "name": "Hello World",
+        "uuid": "151ebc38-659c-4752-b51c-cc7d1d7b1ef4",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 1,
+        "topics": [
+          "strings"
+        ]
```

Comment:
The `topics` key is not actually used anymore. We've deprecated it in v3 in favor of `practices` and `prerequisites`. So you could consider removing the field.